packages/form-js-editor/src/render/components/properties-panel/entries/DefaultValueEntry.js

```diff
@@ -147,7 +150,7 @@ function DefaultValueNumber(props) {
   });
 }
 
-function DefaultValueMulti(props) {
+function DefaultValueSingleSelect(props) {
```

Comment:
@pinussilvestrus Because this component now only makes sense with singleselect type values controls (radio, select). We have plans to also make something available for multiselects (checklist/taglist), but since it requires some properties panel changes and a new control it'll come later.

Lmk if you find something wrong with the reasoning :)

Reply:
Makes sense to me 👍